dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/cdi/SPIMetaData.java

```diff
@@ -23,7 +23,7 @@
 import com.ibm.ws.cdi.extension.CDIExtensionMetadataInternal;
 
 import io.openliberty.cdi.spi.CDIExtensionMetadata;
-import io.openliberty.microprofile.telemetry.internal.helper.AgentDetection;
+import io.openliberty.microprofile.telemetry.common.internal.helper.AgentDetection;
```

Comment:
I would have named this package `io.openliberty.microprofile.telemetry.internal.common` ... no need for the `helper` subpackage.